### PR TITLE
begin PR changelog check

### PR DESCRIPTION
**Pull request by samsalisbury**
### Justification

No JIRA, this is to make sure we keep the changelog up to date.

### Summary

Fail PRs if there's no changelog update.

### Quality

All changes to behavior should be accompanied by relevant tests which both document and protect it.

This PR includes:

  - [ ] New or updated tests which validate the new behavior.  _(Thank you!)_
  - [ ] New or updated behavior which has been manually tested. _(Please provide a link to relevant logs if this is the case.)_
  - [ ] New or updated behavior which is not testable in a reasonable time frame. _(Please ask for help if this is the case, more things are testable than we sometimes think!)_
  - [ ] No new or changed behavior.  _(Just documentation, configuration, or pure refactoring.)_

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+          FILES_URL="/repos/${{ github.repository}}/pulls/$PR_NUMBER/files"
+          CHANGED_FILES_RESPONSE="$(
+            curl \
+              -H "Accept: application/vnd.github.v3+json" \
+              -H 'Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
+              "$FILES_URL"
+          )"
+
+          echo "$CHANGED_FILES_RESPONSE"
```
Comment (marianoasselborn) [inline .github/workflows/pr.yml:33]: Could there be an advantage if putting this into its own script, and using this step to just call it? I see we do that for some functionality under ``/scripts`` already.

Comment (samsalisbury) [reply .github/workflows/pr.yml:33]: Yes, definitely! I'd forgotten about this PR tbh, it should be "draft" as it was only half done iirc.

Comment (samsalisbury) [reply .github/workflows/pr.yml:33]: I'll come back to this soon.

**Comment by samsalisbury** [comment]
Closing as stale.

**Pull request closed without merging**